R/checkers.R

```diff
@@ -45,3 +57,268 @@ check_elements_non_neg <- function(x, arg = "x", call = rlang::caller_env()) {
   }
   invisible()
 }
+
+#' Check there is no missignness in a particular vector
+#'
+#' @param x the vector to check
+#' @param arg the name of the vector to check
+#' @param call Calling environment to be passed to [cli::cli_abort()] for
+#' traceback.
+#'
+#' @return NULL, invisibly
+check_non_missingness <- function(x, arg = "x", call = rlang::caller_env()) {
```

Comment:
For consistency with `checkmate` these functions should be named `assert_` rather than `check_`
```suggestion
assert_non_missingness <- function(x, arg = "x", call = rlang::caller_env()) {
```

Reply:
Ok can go through and change these